flyway/gxa/migrations/V8__gxa-create-exp-design.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE exp_design_column
+(
+    experiment_accession VARCHAR(255) NOT NULL,
+    column_name VARCHAR(255) NOT NULL,
+    is_factor BOOLEAN NOT NULL,
+    column_order INTEGER NOT NULL,
+    id serial NOT NULL
+        CONSTRAINT exp_design_column_sequence_pk
+            PRIMARY KEY,
+    UNIQUE (experiment_accession, column_name, is_factor)
+);
+
+CREATE TABLE exp_design
+(
+    marray VARCHAR(255),
```

Comment:
I'd say this belongs in the other table

Reply:
I guess you mean to have a column which is the array design? So we can remove this and have it be another entry in `exp_design_column`, e.g.:

| experiment_accession | column_name  | is_factor | column_order |     id       |
|----------------------|--------------|-----------|--------------|--------------|
|      E-TABM-713      | Array design | FALSE     | 1            | &lt;UUID&gt; |  

However, unless experiments with multiple microarrays are a possibility I might go so far as to say that we could remove it entirely and store it somewhere else. However I can see that could complicate things a bit and I’m happy to leave it here.

Reply:
Okay, will move this field to the other table.